packages/client/hmi-client/src/components/workflow/ops/compare-datasets/tera-compare-datasets-drilldown.vue

```diff
@@ -209,9 +239,10 @@ const updateCriteria = (card: Partial<CriteriaOfInterestCard>, index: number) =>
 
 const knobs = ref<BasicKnobs>({
 	criteriaOfInterestCards: [],
-	selectedPlotValue: PlotValue.PERCENTAGE,
+	selectedPlotType: PlotValue.PERCENTAGE,
 	selectedCompareOption: CompareValue.IMPACT,
-	selectedDataset: null
+	selectedDataset: null,
+	chartSettings: null
```

Comment:
why chartSettings needs to be inside knobs? In my understanding knobs are for input settings and it should be separated from output settings.